app/controllers/teams_controller.rb

```diff
@@ -44,7 +44,7 @@ def typeahead
     authorize @team
     @query = params[:query]
     matches = User.search_from_query(@team.member_ids, "#{@query}%").pluck(:username)
-    matches = matches.map { |user| { username: user } }
+    matches = matches.map { |user| { name: user } }
```

Comment:
How doesn't this break the form for auto-completing user names in the "Add member" form ?